core/src/test/java/org/elasticsearch/update/UpdateIT.java

```diff
@@ -461,7 +462,7 @@ public void testUpdateRequestWithScriptAndShouldUpsertDoc() throws Exception {
 
     public void testContextVariables() throws Exception {
         assertAcked(prepareCreate("test")
-                        .setSettings("index.mapping.single_type", false)
+                        .setSettings("index.version.created", Version.V_5_6_0.id)
```

Comment:
Maybe worth leaving a comment that we can remove this setting, the parent, and the second document once we no longer support types.